projects/packages/analyzer/src/api/class-analyze-controller.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Automattic\Jetpack\Analyzer;
+
+class Analyze_Controller extends Controller {
+	public function get() {
+		try {
+			Locker::lock();
+		} catch ( \Throwable $th ) {
+			// TODO: extend default Exception class, and use instance_of instead.
+			if ( 'Locked already' === $th->getMessage() ) {
+				return $this->model->get_status();
+			}
+		}
```

Comment:
Did you intend to ignore the exception if the message was something other than "Locked already"?